src/components/donation/Steps/DonateMethods/DonateMethods.tsx

```diff
@@ -23,15 +25,19 @@ export default function DonateMethods({ donaterConfig, state }: Props) {
       className="grid content-start mt-2"
       defaultIndex={state.details?.method === "crypto" ? 1 : 0}
     >
-      <Tab.List as="div" className="grid grid-cols-3 mb-6">
+      <Tab.List
+        as="div"
+        className="grid grid-flow-col sm:grid-cols-4 justify-stretch mb-6"
```

Comment:
Tried making the tab list have an auto-overflow with a scroller on mobile screens, but had difficulties. 
Opted for updating tab widths to fit the mobile screen width and the `grid-flow-col sm:grid-cols-4` was the only one that did the job.
@SovereignAndrey @ap-justin open to suggestions here